onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -127,6 +127,7 @@ std::vector<SupportedOp> supported_op_mode = {
     {"Erf", V_2020_4, {"CPU", "GPU"}},
     {"Exp", V_2020_4, {"CPU", "GPU"}},
     {"Expand", V_2022_1, {"CPU", "GPU"}},
+    {"Expand", V_2024_3, {"CPU", "GPU"}},
```

Comment:
@ankitm3k  The Expand op will be marked as supported since OV 2022_1. Is it not getting marked as supported? 

The change in the no_dimension_supported is valid change.